var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -471,14 +471,22 @@ def setup_run_environment(self, env):
         if self.spec.satisfies("+mpi"):
             env.set("MPICC_CC", self.compiler.cc)
             env.set("MPICXX_CXX", self.compiler.cxx)
+            env.set("CC", self.spec["mpi"].mpicc)
+            env.set("CXX", self.spec["mpi"].mpicxx)
```

Comment:
This looks like those two are swapped? What's the difference?

Reply:
`MPICC_CC` sets the underlying compiler for `mpicc` to the gcc used to install the package instead of the one coming from the system when `neuron` module is loaded and by setting `CC` to `mpicc` we compile with the mpi libraries. Setting `CC` is needed because `nrnivmodl` which compiles the `mod files` looks for the `CC/CXX` variables set in the environment for the compilation of the `mod files`

Reply:
Thanks!